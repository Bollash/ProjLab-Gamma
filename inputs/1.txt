Create Space
Add Asteroid Iron
Add Asteroid Iron
Neighbour 0 1
Add Settler 0
Move 0 0
Status All