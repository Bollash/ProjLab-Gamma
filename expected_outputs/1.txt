Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
A kapott aszteroidák szomszédossá váltak.
Sikeres volt a telepeshozzáadás
Az aktor az aszteroidára mozgott.
Space
1
10
10
False
2
1
Asteroid
0
3
3
3
Iron
1
ast 1
0
Asteroid
1
3
3
3
Iron
1
ast 0
1
0
Settler
0
1
0
0
